spec/controllers/vm_infra_controller_spec.rb

```diff
@@ -65,7 +65,7 @@
 
   let(:custom_attr2) do
     FactoryBot.create(
-      :custom_attribute,
+      :ems_custom_attribute,
```

Comment:
Thanks for the great sleuthing @kbrock. I changed this to use the ems custom attribute factory which sets source to VC.